requirements.txt

```diff
@@ -1,5 +1,6 @@
 pandas==0.23.4
 redis==2.10.6
 ray==0.5.3
+psutil==5.4.8
```

Comment:
`psutil` comes with `ray[debug]` I believe. I'm ok with declaring this explicitly as well.